packages/docs/src/styleguide.md

```diff
@@ -23,9 +25,9 @@ Don't forget to include the main CSS stylesheet too!
 
 ### Beyond core styles
 
-The Blueprint team maintains multiple packages of styles and JavaScript components on NPM (in the scope
-`@blueprintjs`). These are listed in the header under _Releases_. They can be installed with
-similar commands as above. Once you've included any additional package(s) in your application, explore the
+Blueprint is a collection of packages of styles and JavaScript components&mdash;the full list appears in this
```

Comment:
First read I thought the "full list" referred to the "components" and not the "packages". Suggestion: `the full list appears` -> `those packages appear`?